pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1113,6 +1113,18 @@ func (b *MasqueradeBindMechanism) getVifIpByProtocol(proto iptables.Protocol) st
 	}
 }
 
+func (b *MasqueradeBindMechanism) getDstAddressesToDnat(proto iptables.Protocol) (string, error) {
```

Comment:
Could you define this function after the caller ?

Reply:
Moved both of these, Thanks